src/morphology/morphoHelpers/suffixHelpers.js

```diff
@@ -0,0 +1,27 @@
+import { flattenDeep } from "lodash-es";
+
+/**
+ * Creates word forms from a list of given suffixes and a stem.
+ *
+ * @param {string}      stem                The stem on which to apply the suffixes.
+ * @param {string[]}    suffixes            The suffixes to apply.
+ * @param {string}      [appendToStem=""]   Optional material to append between the stem and the suffixes.
+ *
+ * @returns {string[]} The suffixed verb forms.
+ */
+export function applySuffixesToStem( stem, suffixes, appendToStem = "" ) {
+	return suffixes.map( suffix => stem.concat( appendToStem ).concat( suffix ) );
```

Comment:
See my other comment about using the `+` operator instead of `concat` to concatenate strings.
`+` is several times faster.
So:
```suggestion
	return suffixes.map( suffix => stem + appendToStem + suffix );
```